<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SubirArchivoButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Subir Archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubirCarpetaButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Subir Carpeta</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="ArchivosTree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Tus Archivos</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>A1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A2</string>
    </property>
    <item>
     <property name="text">
      <string>Hola</string>
     </property>
     <item>
      <property name="text">
       <string>Hola</string>
      </property>
     </item>
    </item>
   </item>
  </widget>
  <widget class="QPushButton" name="ActualizarButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="AmigosListView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>141</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TusAmigosLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tus Amigos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ConversarButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Conversar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ChatTextField">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Escribe aqui...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnviarButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>590</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QListView" name="ChatListView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ConversandoConLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Conversando con: Nadie</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
